src/shared/jump_to_bottom_button.rs

```diff
@@ -178,3 +232,12 @@ impl JumpToBottomButtonRef {
         }
     }
 }
+
+/// The number of unread messages in the room
+#[derive(Clone, Debug)]
+pub enum UnReadMessageCount {
+    /// Display green badge without text
```

Comment:
the fact that it's green is just an implementation detail that may change.
```suggestion
    /// There are unread messages, but we do not know how many.
```